packages/trader/src/App/Components/Elements/market-countdown-timer.tsx

```diff
@@ -151,7 +152,9 @@ const MarketCountdownTimer = observer(
         let opening_time_banner = null;
         if (opening_time) {
             const formatted_opening_time = convertTimeFormat(opening_time);
-            const target_date = moment(new Date()).add(days_offset, 'days');
+            const target_date = moment(new Date())
+                .locale(getlangFromUrl()?.toLowerCase() ?? 'en')
+                .add(days_offset, 'days');
```

Comment:
@kate-deriv what if we try using `current_language` from `common-store` instead ? :)

```
const { current_language } = common;
```

```suggestion
            const target_date = moment(new Date())
                .locale(current_language.toLowerCase())
                .add(days_offset, 'days');
```

Reply:
Will apply